docs/tutorials/custom/custom-base-dog.md

```diff
@@ -10,7 +10,7 @@ videos:
     "/tutorials/custom-base-dog/base-control-dog.webm",
     "/tutorials/custom-base-dog/base-control-dog.mp4",
   ]
-videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's Control tab on the Viam app open in a browser window."
+videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's CONTROL tab on the Viam app open in a browser window."
```

Comment:
```suggestion
videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the machine's Control tab on the Viam app open in a browser window."
```
In alt text this is read as C O N T R O L which is annoying - so best not to change to all caps there